lib/kuiq/view/busy.rb

```diff
@@ -0,0 +1,73 @@
+require "kuiq/view/global_stats"
+require "kuiq/view/busy_stats"
+require "kuiq/view/footer"
+
+module Kuiq
+  module View
+    class Busy
+      include Glimmer::LibUI::CustomControl
+
+      option :job_manager
+
+      body {
+        vertical_box {
+          global_stats(group_title: t("Summary"), model: job_manager, attributes: Model::Job::STATUSES) {
+            stretchy false
+          }
+
+          busy_stats(group_title: t("Status"), model: job_manager, attributes: %i[process_size total_concurrency busy utilization total_rss])
+
+          group(t("Processes")) {
+            margined false
+            table {
+              text_column(t("Name"))
+              text_column(t("Started"))
+              text_column(t("RSS"))
+              text_column(t("Threads"))
+              text_column(t("Busy"))
+
+              cell_rows <= [job_manager, :process_set, column_attributes: {
+                t("Name") => :identity,
+                t("Started") => :started_at,
+                t("RSS") => :rss,
+                t("Threads") => :concurrency,
+                t("Busy") => :busy
```

Comment:
As long as it passes Standard's formatter. If it's cool with Standard, it's cool with me.

Reply:
Actually, the View layer in Glimmer has its own innovated declarative GUI code conventions because standard Ruby conventions don’t cut it and would make developers write imperative looking code where it needs to read like declarative GUI code. I’d like to eventually codify those Glimmer conventions into a linter plugin (eg for Rubocop) that only runs against the view directory in a Glimmer application. But, in the meantime, I don’t find it an issue to disable the standard checker in the view directory. I personally don’t use linters at all in all my projects because I find they limit creative code organization and innovation at coming up with better organizational structures that could vary from situation to situation even within the same codebase. I find that the reward of using linters is not worth the cost of minimizing the fuzzy human element, and I think the linters’ sales pitch that is advanced by the general software engineering community overblown and exaggerated. I think linters are mostly helpful for software engineering beginners only.